tests/installation/bootloader.pm

```diff
@@ -236,15 +242,15 @@ sub run() {
         $args .= " autoupgrade=1";
     }
 
-    type_string $args, 13;
+    type_string $args, $slower_typing_speed;
     save_screenshot;
 
     if (get_var("FIPS")) {
-        type_string " fips=1", 13;
+        type_string " fips=1", $slower_typing_speed;
         save_screenshot;
     }
 
-    registration_bootloader_params;
+    registration_bootloader_params($slower_typing_speed);
```

Comment:
injecting the variable into the helper sub and set it to "4" instead of "13" in before as this is where the real problem occurred